src/components/ScrollOnRender/ScrollOnRender.tsx

```diff
@@ -1,14 +1,17 @@
-import type { PropsWithChildren } from "react";
-
-import { useScrollOnRender } from "hooks/useScrollOnRender";
+import { useEffect, type PropsWithChildren } from "react";
 
 type Props = {
   scrollArea?: HTMLElement | null;
 } & PropsWithChildren;
 
 const ScrollOnRender = ({ children, scrollArea }: Props) => {
-  const onRenderRef = useScrollOnRender(scrollArea);
-  return <div ref={onRenderRef}>{children}</div>;
+  useEffect(() => {
+    if (scrollArea) {
+      scrollArea.scrollIntoView();
```

Comment:
It might be nice to set `{ behavior: "smooth" }`.

Reply:
Done! Although it is strange that, on my laptop running Ubuntu 22.04, the scrolling animation is not smooth in Chrome.